include/common/arch/mips/asm/bitsperlong.h

```diff
@@ -0,0 +1,10 @@
+#ifndef __CR_BITSPERLONG_H__
+#define __CR_BITSPERLONG_H__
+
+#ifdef CONFIG_MIPS
+# define BITS_PER_LONG 64
+#else
+# define BITS_PER_LONG 32
```

Comment:
Are we going to support mips32? Even if yes, the 32-bit version of `BITS_PER_LONG` can be added then.